doc/gvmd.8.xml

```diff
@@ -405,23 +412,29 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
     </p>
   </section>
 
-  <section name="MORE INFORMATION ABOUT THE OpenVAS PROJECT">
+  <section name="MORE INFORMATION ABOUT Greenbone Vulnerability Management">
     <p>
       The canonical places where you will find more information
-      about the OpenVAS project are:
-
-      <url href="http://www.openvas.org/"/>
-        (Official site)
-
-      <url href="http://wald.intevation.org/projects/openvas/"/>
+      about the Greenbone Vulnerability Manager are:
+    </p>
+    <p>
+      <url href="https://community.greenbone.net"/>
+        (Community portal)
+    </p>
+    <p>
+      <url href="https://github.com/greenbone"/>
         (Development Platform)
     </p>
+    <p>
+      <url href="https:/greenbone.net"/>
```

Comment:
``https:/greenbone.net`` -> ``https://greenbone.net``

Reply:
I've fixed this in 30167e42be23fa6c218abc80964f26206c36a038